css/pages/contact.scss

```diff
@@ -27,7 +27,7 @@
         padding-top:30px;
         border-radius: 50%;
         vertical-align: middle;
-        margin-right:30px;
+        margin-left:30px;
```

Comment:
I think the margin-left here is throwing off the centering, like the icons don't seem to be horizontally centered because of it.  I would suggest doing a margin on both sides to combat this problem.  So for example, instead of `margin-left` do

```scss
margin: 0 15px;
```